awsx/lb/networkLoadBalancer.ts

```diff
@@ -89,7 +90,7 @@ export class NetworkLoadBalancer extends schema.NetworkLoadBalancer {
       {
         vpcId: this.vpcId,
         protocol: "TCP",
-        port: 80,
+        port: defaultTargetGroupPort ?? 80,
         ...defaultTargetGroup,
```

Comment:
Ok, just revisiting the top level default port and digging in further ...

I see there's an existing `defaultTargetGroup. All properties on this are optional and are just spread after our defaults as overrides. 

If both the `defaultTargetGroup.port` and `defaultTargetGroupPort` are specied, with different values what should we do?

Currently we would just take the `defaultTargetGroup.port` over `defaultTargetGroupPort`. Perhaps we should error instead as this is certainly a mistake?